actors/common/src/main/java/org/sunbird/common/quartz/scheduler/ShadowUserMigrationScheduler.java

```diff
@@ -231,19 +231,12 @@ private void updateUser(String processId,MigrationUser migrationUser, ShadowUser
      * ALSO this method will allow only CLAIMED user record to be updated in diksha db.
      * @param processId
      * @param migrationUser
-     * @param shadowUser
+     * @param existingShadowUser
      */
-    private void updateUserInShadowDb(String processId,MigrationUser migrationUser, ShadowUser shadowUser) {
-        if(!isSame(shadowUser,migrationUser)){
-            Map<String, Object> propertiesMap = new WeakHashMap<>();
-            propertiesMap.put(JsonKey.EMAIL,migrationUser.getEmail());
-            propertiesMap.put(JsonKey.PHONE,migrationUser.getPhone());
-            propertiesMap.put(JsonKey.PROCESS_ID,processId);
-            propertiesMap.put(JsonKey.NAME, migrationUser.getName());
-            propertiesMap.put(JsonKey.ORG_EXT_ID, migrationUser.getOrgExternalId());
-            propertiesMap.put(JsonKey.UPDATED_ON, new Timestamp(System.currentTimeMillis()));
-            propertiesMap.put(JsonKey.USER_STATUS,getInputStatus(migrationUser.getInputStatus()));
-            if (shadowUser.getClaimStatus() != ClaimStatus.CLAIMED.getValue() && shadowUser.getClaimStatus() != ClaimStatus.REJECTED.getValue() && shadowUser.getClaimStatus() != ClaimStatus.FAILED.getValue()) {
+    private void updateUserInShadowDb(String processId,MigrationUser migrationUser, ShadowUser existingShadowUser) {
+        if(!isSame(existingShadowUser,migrationUser)){
```

Comment:
This condition will still fail - here;s the scenario.
1. User A was uploaded and A rejected the claim unknowningly.
2. Admin reuploaded.
isSame = true and therefore !isSame = false.

Reply:
now checking status only for claimed-users.